src/ext/java/org/opentripplanner/ext/actuator/ActuatorAPI.java

```diff
@@ -87,11 +92,19 @@ public Response health(@Context OtpServerRequestContext serverContext) {
    */
   @GET
   @Path("/prometheus")
-  public Response prometheus(@Context PrometheusMeterRegistry prometheusRegistry) {
+  @Produces({ TextFormat.CONTENT_TYPE_004, TextFormat.CONTENT_TYPE_OPENMETRICS_100 })
+  public Response prometheus(
+    @Context final PrometheusMeterRegistry prometheusRegistry,
+    @HeaderParam(ACCEPT) final String acceptHeader
+  ) {
+    final var contentType = acceptHeader.contains("application/openmetrics-text")
```

Comment:
Can `acceptHeader` ever be null?

Reply:
Can you add either the `Nullable` or `NotNull` annotation to make it clear?

Reply:
The `acceptHeader` seems to never be `null`: even if `Accept` is not provided I had no issue during testing. But I'll add a `@DefaultValue("*/*")` for good measure!